dockerspawner/dockerspawner.py

```diff
@@ -196,15 +203,10 @@ def volume_mount_points(self):
         all the locations where you're going to mount volumes when you call
         create_container.
 
-        Returns a list of all the values in self.volumes or
+        Returns a sorted list of all the values in self.volumes or
         self.read_only_volumes.
         """
-        return list(
-            itertools.chain(
-                self.volumes.values(),
-                self.read_only_volumes.values(),
-            )
-        )
```

Comment:
Now that `itertools` is not used in this file, can you take out the import from above to satisfy pyflakes?